### ISSUE TITLE
Lists: Christmas tree recycling page

### ISSUE THREAD

**Issue by chasechenevert** [cityofaustin/techstack#3437]
From Annie
>We can discuss Austin Resource Recovery's Christmas tree recycling page when sprint 44 starts. Thoughts on a temporary solution for the list under "More free drop-off sites", in the interim before we can start aggregating locations on a map? Maybe formatting similar to the table in the section below? Here's the preview https://alpha.austin.gov/en/preview/information/UGFnZVJldmlzaW9uTm9kZToyMjQx

**Comment by chasechenevert** [comment]
CC: @annieszot @desigonz 
## Issue:
Formatting currently doesn’t look like a list because all lines are new paragraphs `<p>`, therefore inheriting 17px of spacing above and below it so none of the information appears to be grouped which isn't a pleasent reading experience.

## Solution: no new code needed!
- Bold the names of locations to support scanning the content
- Forced line breaks are available in Joplin, so after entering the name of the location on the same line press **shift + enter** on your keyboard to insert a line break within the same paragraph **<p>** element. 

**Before**
<img width="657" alt="Screen Shot 2019-11-20 at 11 42 25 AM" src="https://user-images.githubusercontent.com/7027270/69263441-f4486900-0b8a-11ea-8ba4-0cf47e9125b4.png">

**After**
<img width="721" alt="Screen Shot 2019-11-20 at 11 42 12 AM" src="https://user-images.githubusercontent.com/7027270/69263456-fb6f7700-0b8a-11ea-9f53-d77a173eed1b.png">

----

We may want to let other content strategists know that this option is available and the situation in which we're using it, and maybe add to the style guide. 

💡 I wonder if it would be helpful in some way to visualize a line break within the paragraph element in Joplin because it's very subtle and difficult to distinguish.

**Comment by chasechenevert** [comment]
@annieszot gonna leave this issue open for you in the review pipeline, so when you've got what you need from it feel free to close it.

**Comment by annieszot** [comment]
Thanks @chasechenevert ! VERY helpful. Will copy info from this issue and make a new issue to add it to the Style Guide. Will inform the content team. 
Informed content team in Slack channels #odd-flock-content and DMs groups with all content strategists
